src/purifier/.gitignore

```diff
@@ -0,0 +1 @@
+dist
```

Comment:
Can this go into the root level `.gitignore` file? We already ignore other inner-level `dist` directories in there.

https://github.com/ampproject/amphtml/blob/aa46011ee501c2f6efcc614594a68e33ed6d3741/.gitignore#L17

Reply:
Good point, done.